packages/jest-snapshot/src/utils.ts

```diff
@@ -136,6 +136,21 @@ export const removeExtraLineBreaks = (string: string): string =>
     ? string.slice(1, -1)
     : string;
 
+export const removeLinesBeforeExternalMatcherTrap = (stack: string): string => {
```

Comment:
I'd like to see some unit tests for this, although ut looks pretty straightforward

Reply:
Sure, I'll see what I can do